Miscellaneous/Problems/kth-prime factor.cpp

```diff
@@ -0,0 +1,46 @@
+
+/* To find the  k-th prime factor of a given number
+
+//Example : n=12 ,  k=3
+  Output  : 3
+*/
+
+
+
+#include<iostream>
+
+int kPrimeFactor(int n, int k)
+{
+    while (n%2 == 0)
+    {
+        k--;
```

Comment:
Use meaningful variable names. Using `i` or `k` is not viable for anyone who is trying to understand your code.